regression/esbmc/github_709-1/test.desc

```diff
@@ -1,4 +1,4 @@
-KNOWNBUG
+CORE
 main.c
 
 ^VERIFICATION FAILED$
```

Comment:
Can you also add the safe version of this regression test?

Reply:
done